### PR TITLE
adding support for website_metadata

### PR DESCRIPTION
**Pull request by shanyas10**
adding metadata processor for website descriptions. adding tests accordingly

### CODE REVIEW

File: bsmetadata/metadata_processors.py [hunk 1]
```diff
@@ -103,10 +103,18 @@ def process_global(self, metadata_attrs: Dict[str, Any]) -> Optional[str]:
         # Example: "foo.bar/Year 2021/" instead of "foo.bar/Year%202021/".
         return "".join([metadata_attrs["key"], self.cfg.metadata_key_value_sep, unquote_plus(metadata_attrs["value"])])
 
+class WebsiteDescriptionProcessor(MetadataProcessor):
+    """An example metadata processor for URLs."""
```
Comment (SaulLu) [inline bsmetadata/metadata_processors.py:107]: A very small change suggestion :slightly_smiling_face: 
```suggestion
    """An example metadata processor for website descriptions."""
```

**Review by SaulLu — APPROVED** [review]
Great! Thanks a lot for the addition, it looks great!

File: bsmetadata/metadata_processors.py [hunk 2]
```diff
@@ -128,10 +128,18 @@ def process_global(self, metadata_attrs: Dict[str, Any]) -> Optional[str]:
         # Example: "foo.bar/Year 2021/" instead of "foo.bar/Year%202021/".
         return "".join([metadata_attrs["key"], self.cfg.metadata_key_value_sep, unquote_plus(metadata_attrs["value"])])
 
+class WebsiteDescriptionProcessor(MetadataProcessor):
+    """An example metadata processor for website descriptions."""
+
+    def process_global(self, metadata_attrs: Dict[str, Any]) -> Optional[str]:
+        # Example: "website_description: BBC is a news organization".
+        return "".join([metadata_attrs["key"], self.cfg.metadata_key_value_sep, metadata_attrs["value"]])
```
Comment (timoschick) [inline bsmetadata/metadata_processors.py:136]: I think I'd personally favour a variant where "Website Description" is written in plain English (instead of "website_description") - that is, ` return "".join("Website Description", self.cfg.metadata_key_value_sep, metadata_attrs["value"]])` - but I guess that's just a matter of taste :)

**Review by timoschick — APPROVED** [review]
Looks good to me, thanks a lot 👍 
Nitpick: It seems some of your formatting does not match our style guide, so you'll probably have to run `make style && make quality` before you can merge this pull request.

File: bsmetadata/metadata_processors.py (hunk 2 above)
Comment (shanyas10) [reply bsmetadata/metadata_processors.py]: Now that you mention it, I too prefer "Website Description" :) Added the change and modified tests accordingly!
